src/index.js

```diff
@@ -86,6 +86,26 @@ class Keyv extends EventEmitter {
 			.then(() => true);
 	}
 
+	has(key) {
+		key = this._getKeyPrefix(key);
+		const store = this.opts.store;
+
+		return Promise.resolve()
+			.then(() => {
+				return store.has(key);
```

Comment:
You would need to implement the `has` method on all official storage adapters before this would work.